From c86645dde30fbd54ae0f187e4428a09ad75e0ba8 Mon Sep 17 00:00:00 2001
From: Peter Oliver <git@mavit.org.uk>
Date: Mon, 15 Jul 2024 12:03:47 +0100
Subject: [PATCH] Fix intermittent failure of dired-test-bug27243-02

* test/lisp/dired-tests.el (dired-test-bug27243-02): Exclude free disk
space from dired listing in this test, in case it changes while it's
running and confuses the result.
---
 test/lisp/dired-tests.el | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/lisp/dired-tests.el b/test/lisp/dired-tests.el
index 2589281e5e1..ea47043dfc5 100644
--- a/test/lisp/dired-tests.el
+++ b/test/lisp/dired-tests.el
@@ -189,7 +189,9 @@ dired-test-bug27243-01
 (ert-deftest dired-test-bug27243-02 ()
   "Test for https://debbugs.gnu.org/cgi/bugreport.cgi?bug=27243#28 ."
   (ert-with-temp-directory test-dir
-    (let ((dired-auto-revert-buffer t) buffers)
+    (let ((dired-auto-revert-buffer t)
+          (dired-free-space nil)
+          buffers)
       ;; On MS-Windows, get rid of 8+3 short names in test-dir, if the
       ;; corresponding long file names exist, otherwise such names trip
       ;; string comparisons below.
-- 
2.45.2

